src/shared/Microsoft.Git.CredentialManager/Authentication/BasicAuthentication.cs

```diff
@@ -27,7 +32,7 @@ public GitCredential GetCredentials(string resource, string userName)
 
             EnsureTerminalPromptsEnabled();
 
-            Context.Terminal.WriteLine("Enter credentials for '{0}':", resource);
+            Context.Terminal.WriteLine("Enter basic credentials for '{0}':", resource);
```

Comment:
While I like "Enter GitHub credentials", I'm not sure what to think about "Enter basic credentials".

Did you consider simply "Log in to `{0}`" or similar?

Reply:
I did consider this.. but this is similar to the message that Git itself prompts and I wanted to make this distinct (because Git will do a prompt if the helper fails).

This raises an interesting question.. do we want GCM-based TTY prompts to start with a prefix to make it really obvious?

```
warning: Something important to tell you!
Enter your GitHub credentials:
Username: john.doe
Password: ******
```
vs
```
gcm: warning: Something important to tell you!
gcm: Enter your GitHub credentials:
gcm: Username: john.doe
gcm: Password: ******
```

Reply:
Makes sense on the messaging. I really like prefixing the lines with who's talking @mjcheetham

Reply:
https://github.com/microsoft/Git-Credential-Manager-Core/issues/63

Created an issue to track this idea of prefixing 'who is speaking'.